modules/undo.py

```diff
@@ -1,13 +1,20 @@
+
+from PyQt6.QtCore import *
+
+from PyQt6.QtWidgets import *
+from modules.object import *
+
 class Undo:
     def __init__(self, parameter):
         self.parameter = parameter
 
-    def redo(self):
-        # Implement the operation that this command represents
-        print(self.parameter+'redo')
 
-    def undo(self):
+    def undo(self, editor):
         # Implement the undo operation for this command
-        print(self.parameter+'undo')
+        if self.parameter['action']=='move':
+            print('move obj to old location')
```

Comment:
can u take out the print statements if ur done with the debugging

Reply:
Done